### PR TITLE
Switching back to webpack.

### PR DESCRIPTION
**Pull request by jgreen01**
- Updating lambda template for the API changes made in node4.3 update.
- Updating build scripts to allow multiple lambdas to be processed at once.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (zjr) [inline .babelrc]: extra line?

File: bin/deploy.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 npm run clean && \
 npm run lint && \
-npm run rollup && \
+npm run webpack && \
 npm run test && \
 npm run gulp && \
-rm -rf tmp
+rm -rf tmp
```
Comment (zjr) [inline bin/deploy.sh]: Would you turn on auto newline at EOF so we don't get this noise?

File: config/templates/lambda.tpl.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export function handler(event, context) {
+export function handler(event, context, callback) {
     console.log(event);
-    context.success('done!');
-}
\ No newline at end of file
+    callback(null, 'done!');
```
Comment (zjr) [inline config/templates/lambda.tpl.js:3]: was this part supposed to change?

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    resolve: {
+        extensions: [ '', '.js','.json' ]
+    },
+    module: {
+        noParse: /node_modules\/json-schema\/lib\/validate\.js/,
+        loaders: [
+            {
+                test: /\.js$/,
+                loader: 'babel-loader',
```
Comment (zjr) [inline webpack.config.js:23]: is babel necessary?

File: config/templates/lambda.tpl.js (hunk 3 above)
Comment (jgreen01) [inline config/templates/lambda.tpl.js:3]: Yep amazon changed their API when they updated to node4.3.

[Transitioning Lambda Function Code to Node.js Runtime v4.3](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-using-old-runtime.html#transition-to-new-nodejs-runtime)

Comment (zjr) [inline config/templates/lambda.tpl.js:3]: Hah that sure didn’t last long

Comment (zjr) [inline config/templates/lambda.tpl.js:3]: They're still two major versions behind though!  is 4.3 really the latest you can use?

File: webpack.config.js (hunk 4 above)
Comment (jgreen01) [inline webpack.config.js:23]: Only if we want be able to use all es6 features in our lambda functions. Hmm node4 looks like it has ok es6 support. ¯_(ツ)_/¯ Up to you.

[ECMAScript 6 compatibility table](https://kangax.github.io/compat-table/es6/#node)

File: bin/deploy.sh (hunk 2 above)
Comment (jgreen01) [inline bin/deploy.sh]: I didn't change the file. The original author wrote that.

**Comment by zjr** [comment]
LGTM

**Pull request merged**